app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -229,11 +229,7 @@ def group_targets_by_ems(targets)
       def preprocess_targets_manager_refresh
         @targets_by_ems_id.each do |ems_id, targets|
           ems = @ems_by_ems_id[ems_id]
-
-          if targets.any? { |t| t.kind_of?(ExtManagementSystem) }
-            targets_for_log = targets.map { |t| "#{t.class} [#{t.name}] id [#{t.id}] " }
-            _log.info("Defaulting to full refresh for EMS: [#{ems.name}], id: [#{ems.id}], from targets: #{targets_for_log}") if targets.length > 1
-          end
```

Comment:
This logic is handled down here https://github.com/ManageIQ/manageiq/blob/e8ee67acf827f2e51e619e59cf8379f00caaeb4d/app/models/manageiq/providers/base_manager/refresher.rb#L262 so this isn't needed anymore